src/Sarif/Schemata/sarif-schema.json

```diff
@@ -487,6 +487,137 @@
             }
         },
 
+        "externalProperties": {
+            "type": "object",
+            "properties": {
+
+                "schema": {
+                    "description": "The URI of the JSON schema corresponding to the version of the external property file format.",
+                    "type": "string",
+                    "format": "uri"
+                },
+
+                "version": {
+                    "description": "The SARIF format version of this log file.",
+                    "enum": [ "2.0.0-csd.2.beta.2019-02-20" ]
+                },
+
+                "guid": {
+                    "description": "A stable, unique identifer for the external properties in the form of a GUID.",
+                    "type": "string"
+                },
+
+                "runGuid": {
+                    "description": "A stable, unique identifer for the external properties in the form of a GUID.",
+                    "type": "string"
+                },
+
+                "conversion": {
+                    "description": "A conversion object that describes how a converter transformed an analysis tool's native reporting format into the SARIF format.",
+                    "$ref": "#/definitions/conversion"
+                },
+
+                "graphs": {
+                    "description": "A dictionary, each of whose keys is the id of a graph and each of whose values is a 'graph' object with that id.",
+                    "type": "object",
+                    "additionalProperties": {
+                        "$ref": "#/definitions/graph"
+                    }
+                },
+
+                "externalizedProperties": {
+                    "description": "Key/value pairs that provide additional information about the run.",
+                    "$ref": "#/definitions/propertyBag"
+                },
+
+                "artifacts": {
+                    "description": "An array of artifact objects relevant to the run.",
+                    "type": "array",
+                    "minItems": 0,
+                    "uniqueItems": true,
+                    "items": {
+                        "$ref": "#/definitions/artifact"
+                    }
+                },
+
+                "invocations": {
+                    "description": "Describes the invocation of the analysis tool.",
```

Comment:
>Describes the invocation of the analysis tool. [](start = 36, length = 46)

'Describes an invocation of the analysis tool that will be merged with an external run.'